web/styles/image_upload_widget.css

```diff
@@ -8,6 +8,12 @@
     .image-block {
         background-size: contain;
         height: 100%;
+
+        img {
+            /* TODO - I'm not sure why this is necessary for the
+               realm icon but not the avatar */
```

Comment:
Obviously not meant to be merged. I was wondering why this might be, and if it's just okay to put this here.

(Also I just realized that I made the `image-block-image` class and maybe I should use that here instead.)

Reply:
It seems OK to have styles here. A few requests:
- Always have a CSS class rather than element type; element types have (minor) performance downsides and are harder to read/audit/edit. 
- Make the comment explain what problem it's solving / failure mode it's preventing.